locales/app.yml

```diff
@@ -610,3 +610,19 @@ _version: 2
   en: "Windows Update"
   es: "Actualización de Windows"
   zh_TW: "Windows 更新"
+"Microsoft Store":
+  en: "Microsoft Store"
+  es: "Tienda de Microsoft"
+  zh_TW: "Microsoft 店鋪"
```

Comment:
```suggestion
  zh_TW: "Microsoft Store"
```
Looks like "Microsoft Store" won't be translated:

> text in the below picture: Microsoft Store Support

<img width="132" alt="Screenshot 2024-10-22 at 9 35 43 PM" src="https://github.com/user-attachments/assets/720b2b31-2e19-4ba3-9093-3a97557f1784">